recipes/gto/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "gto" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gto-{{ version }}.tar.gz
+  sha256: ce9501b6a7e283e331239fb9c957ff0170ff48df1c8402431309138c5d462556
+
+build:
+  entry_points:
+    - gto = gto.cli:app
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - setuptools-scm
+    - fastentrypoints >=0.12
+    - pip
+  run:
+    - python >=3.6
```

Comment:
you seem to be running an outdated version of `grayskull`. i suggest updating your version and running the feedstock generation again.

Reply:
re-ran grayskull with version 1.8.4